### DISCUSSION TITLE
How to disabled converting rgba color values to hexadecimal color values

### DISCUSSION THREAD

**Discussion by martSforever** [nuxt/nuxt#17232] [Questions]
- I encountered a problem. Huawei's own browser does not support hexadecimal alpha color values;
- I have css code like this "background-color: rgba(0, 0, 0, 0.3);", It's not change on development mode, and it's work on Huawei's own browser; but after building project It turns to "background-color: #0000004d;" and not work in Huawei's own browser;
- How can i fixed this problem or disabled convert rgba color values to hexadecimal color values;

**Comment by mirumirumi** [comment]
Hi!
How about trying the HSL format? like this:
`hsl(260deg 2% 25%)`
